frontend/installation-ui/web/micro-ui-internals/packages/modules/pm/src/pages/employee/CreateProject.js

```
@@ -564,9 +580,26 @@ const CreateProject = () => {
-        await upsertProject(newProjectFormData);
+        const savedData = {
+          projectDetails: {
+            projectType: projectTypeData?.filter((projectType) => projectType?.name === createdProject?.projectType)?.[0],
+            justificationCode: createdProject?.additionalDetails?.justificationCode,
+            projectDuration: {
+              startDate: formatDate(createdProject?.startDate),
+              endDate: formatDate(createdProject?.endDate),
+            }
+          },
+          geographyDetails: createdProject?.additionalDetails.geographyDetails,
+        }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)